src/olympia/bandwagon/models.py

```diff
@@ -30,7 +30,10 @@ class Collection(ModelBase):
     name = TranslatedField(require_locale=False)
     slug = models.CharField(max_length=30, blank=True, null=True)
 
-    description = NoLinksNoMarkupField(require_locale=False)
+    # biography can (and sometimes does) contain html and other unsanitized
```

Comment:
Should this comment say "description" as opposed to "biography"?

Reply:
Oops, yeah

Reply:
649f7fb8b508b50e05c1644fba2e69124a565769